lib/asm-parser-vc.js

```diff
@@ -129,6 +129,49 @@ class AsmParser extends AsmParserBase {
 
         let seenEnd = false;
 
+        let datadefLabels = [];
```

Comment:
I think these can both be `const` (as we don't reassign them. Javascript has a very funny idea of const-ness)

Reply:
indeed, changed it